### PR TITLE
Feature/xbox teleop usb check

### PR DESCRIPTION
**Pull request by hello-fazil**
This PR introduces a polling step to the [stretch_xbox_controller_teleop.py](https://github.com/hello-robot/stretch_body/blob/feature/xbox_teleop_usb_check/tools/bin/stretch_xbox_controller_teleop.py#L385:L414) script which waits for all the hello* USB devices to be found under `/dev` directory before initiating a Robot() class object.

### CODE REVIEW

File: tools/bin/stretch_xbox_controller_teleop.py [hunk 1]
```diff
@@ -371,33 +380,69 @@ def manage_calibration(robot,controller_state):
             print('press the start button to calibrate the robot')
         else:
             first_home_warn = False
+
+
+########################### Check and wait for USB Devices ###########################
+
+def check_usb_devices(wait_timeout=5):
```
Comment (hello-binit) [inline tools/bin/stretch_xbox_controller_teleop.py:387]: Should return boolean `all_found`.

File: tools/bin/stretch_xbox_controller_teleop.py [hunk 2]
```diff
@@ -371,33 +380,69 @@ def manage_calibration(robot,controller_state):
...
+use_head_mapping = True
+use_dex_wrist_mapping = False
+use_stretch_gripper_mapping = True
+
 
 def main():
     global use_head_mapping, use_dex_wrist_mapping, use_stretch_gripper_mapping
     xbox_controller = xc.XboxController()
     xbox_controller.start()
+    check_usb_devices(wait_timeout=5)
```
Comment (hello-binit) [inline tools/bin/stretch_xbox_controller_teleop.py:427]: If this method fails, let's emit some warning to the user. Maybe beep 3 times? @aedsinger

**Review by hello-binit — CHANGES_REQUESTED** [review]

File: tools/bin/stretch_xbox_controller_teleop.py (hunk 1 above)
Comment (hello-fazil) [reply tools/bin/stretch_xbox_controller_teleop.py:387]: Done!

**Review by hello-binit — APPROVED** [review]
We can add a triple beep later if it seems necessary. Otherwise, I think these changes are good to merge!

**Pull request merged**